### ISSUE TITLE
Is RipMe not intended to download all files in a sub-reddit?

### ISSUE THREAD

**Issue by Zachary24** [RipMeApp/ripme#1042]
* Ripme version: 1.7.69
* Java version: "11.0.1" 2018-10-16
* Operating system: MacOS 10.14

## Expected Behavior

Download all images (in a subreddit) from now till' beginning of the sub.

## Actual Behavior

Noticed some images/videos from the sub weren't ripped, re-ripped, still nothing... Used PSAW to grab all imgur/gfycat files from the sub, then used wget to download them, a little of 25,000 images/videos, compared to ripme which only ripped about 9,000 files.. I plan to do some more testing with other subs.

**Comment by cyian-1756** [comment]
> Is RipMe not intended to download all files in a sub-reddit?

It is intended to download all files

**Comment by Hrxn** [comment]
> Noticed some images/videos from the sub weren't ripped, re-ripped, still nothing... Used PSAW to grab all imgur/gfycat files from the sub, then used wget to download them, a little of 25,000 images/videos, compared to ripme which only ripped about 9,000 files.. I plan to do some more testing with other subs.

What do you mean with PSAW?
This? `https://github.com/dmarx/psaw`

**Comment by Zachary24** [comment]
Yes, PSAW connects to the PushShiftAPI which then connects to Reddit, puts all the urls in a csv, wget downloads them, then the script would delete any files that aren't, .jpg .jpeg .mp4 .gif .png..

**Comment by throwaway231** [comment]
Hey! Would it be possible to get your PSAW script?

**Comment by Zachary24** [comment]
Are you sure? the script itself doesn't store the download history, doesnt automatically remove the other csv rows as well as downloads some random .html files (As it was quickly made just to test it). After seeing the difference in ripme and the basic script I decided to start building my own "ripme" alternative, which right now supports fully supports Reddit, Instagram, Wget, and youtube-dl.. Somewhat supports Imgur albums and Wikipedia Tables, and more to come! So if you're interested in that I could let you know when I'm finished.

**Comment by lpanhirsch** [comment]
Nice! Please let me know

**Comment by throwaway231** [comment]
I'm very interested! Please lmk!

**Comment by ealgase** [comment]
Ripme only rips 1000 entries which is what the Reddit api returns.

**Comment by j4sonshu** [comment]
> > I'm very interested! Please lmk!
> 
> https://github.com/Zachary24/RipPy
> 
> Its not been released yet, its just the github page. It's almost finished though, all I really have left until the release is creating the basic google search image ripper, and cleaning up instagram youtube and reddit!

Hey, did you ever finish RipPy? Would be very interested in trying it out as an alternative to RipMe.
